youtube_dl/extractor/nbc.py

```diff
@@ -109,18 +109,18 @@ class NBCSportsVPlayerIE(InfoExtractor):
     _VALID_URL = r'https?://vplayer\.nbcsports\.com/(?:[^/]+/)+(?P<id>[0-9a-zA-Z_]+)'
 
     _TESTS = [{
-        'url': 'https://vplayer.nbcsports.com/p/BxmELC/nbcsports_share/select/9CsDKds0kvHI',
+        'url': 'http://vplayer.nbcsports.com/p/BxmELC/chicago_embed/select/media/u4xMi3ngFKsS',
```

Comment:
https://vplayer.nbcsports.com/p/BxmELC/nbcsports_embed/select/9CsDKds0kvHI is still available.